crates/store/re_protos/proto/rerun/v1alpha1/catalog.proto

```diff
@@ -27,6 +28,14 @@ message FindEntriesResponse {
   repeated EntryDetails entries = 1;
 }
 
+// DeleteDatasetEntry
+
+message DeleteEntryRequest {
+  rerun.common.v1alpha1.Tuid id = 1;
```

Comment:
I would prefer if we had an actual `EntryId` type. Our total number of ID things has quadrupled in the last weeks, a lot of things are Tuids now...